assets/js/job-editor/mount.tsx

```diff
@@ -1,230 +0,0 @@
-import pDebounce from 'p-debounce';
-import pRetry from 'p-retry';
-import { createRoot } from 'react-dom/client';
-import { EDITOR_DEBOUNCE_MS } from '../common';
-import type { LiveSocket } from 'phoenix_live_view';
-import type {
-  PhoenixHook,
-  GetPhoenixHookInternalThis,
-} from '../hooks/PhoenixHook';
-import { sortMetadata } from '../metadata-loader/metadata';
-import type { Lightning } from '../workflow-diagram/types';
-import type WorkflowEditorEntrypoint from '../workflow-editor';
-import type { WorkflowStore } from '../workflow-editor/store';
-import type JobEditor from './JobEditor';
-
-declare global {
-  interface Window {
-    jobEditor?: GetPhoenixHookInternalThis<JobEditorEntrypoint> | undefined;
-  }
-}
-
-type JobEditorEntrypoint = PhoenixHook<
-  {
-    componentRoot: ReturnType<typeof createRoot> | null;
-    changeEvent: string;
-    field?: HTMLTextAreaElement | null;
-    findWorkflowEditorStore(): Promise<WorkflowStore>;
-    workflowEditorStore: WorkflowStore;
-    handleContentChange(content: string): Promise<void>;
-    metadata?: true | object;
-    observer: MutationObserver | null;
-    render(): void;
-    requestMetadata(): Promise<{}>;
-    setupObserver(): void;
-    pushChange(content: string): Promise<void>;
-    _debouncedPushChange(content: string): Promise<void>;
-    currentContent: string;
-  },
-  {
-    adaptor: string;
-    source: string;
-    disabled: string;
-    disabledMessage: string;
-    jobID: string;
-  }
->;
-
-type AttributeMutationRecord = MutationRecord & {
-  attributeName: string;
-  oldValue: string;
-};
-
-let JobEditorComponent: typeof JobEditor | undefined;
-
-export default {
-  findWorkflowEditorStore(): Promise<WorkflowStore> {
-    return pRetry(
-      attempt => {
-        console.debug('Looking up WorkflowEditorHook', { attempt });
-        return lookupWorkflowEditorHook(this.liveSocket, this.el);
-      },
-      { retries: 5 }
-    ).then(hook => {
-      return hook.workflowStore;
-    });
-  },
-  reconnected() {
-    console.debug('Reconnected JobEditor');
-    // this.handleContentChange(this.currentContent);
-  },
-  mounted() {
-    let event = 'editor load';
-    let start = instrumentStart(event);
-
-    window.jobEditor = this;
-
-    this._debouncedPushChange = pDebounce(this.pushChange, EDITOR_DEBOUNCE_MS);
-
-    import('./JobEditor').then(module => {
-      JobEditorComponent = module.default as typeof JobEditor;
-      this.componentRoot = createRoot(this.el);
-
-      const { changeEvent } = this.el.dataset;
-      if (changeEvent) {
-        this.changeEvent = changeEvent;
-      } else {
-        console.warn('Warning: No changeEvent set. Content will not sync.');
-      }
-      this.setupObserver();
-      this.render();
-
-      let end = instrumentEnd(event);
-
-      this.pushEventTo(this.el, 'job_editor_metrics_report', {
```

Comment:
Have we lost this event? Do we need it still?

Reply:
Just seen it in the other PR, so I think we're good once that's merged in

Reply:
Yeah. we wouldn't lose any event. I just ignored it here because I had harder problems to deal with 😂